vision_datasets/image_classification/classification_as_kvp_dataset.py

```diff
@@ -70,20 +55,79 @@ def __init__(self, classification_dataset: VisionDataset):
         super().__init__(dataset_info, dataset_manifest, dataset_resources=classification_dataset.dataset_resources)
 
     def construct_schema(self, class_names: typing.List[str]) -> typing.Dict[str, typing.Any]:
-        schema: typing.Dict[str, typing.Any] = BASE_CLASSIFICATION_SCHEMA  # initialize with base schema
-        schema["fieldSchema"][f"{CLASS_NAME_KEY}"]["classes"] = {c: {"description": f"A single class name. Only output {c} as the class name if present."} for c in class_names}
+        raise NotImplementedError
+
+    def construct_kvp_label_data(self, label_names: typing.List[str]) -> typing.Dict[str, typing.Union[typing.Dict[str, typing.Dict[str, str]], None]]:
+        raise NotImplementedError
+
+
+class MultiClassAsKeyValuePairDataset(ClassificationAsKeyValuePairDatasetBase):
+    CLASS_NAME_KEY = "className"
+    SCHEMA_BASE = {
+        "name": "Multiclass image classification",
+        "description": "Classify images into one of the provided classes.",
+        "fieldSchema": {
+            f"{CLASS_NAME_KEY}": {
+                "type": "string",
+                "description": "Class name that the image belongs to.",
+                "classes": {}
+            }
+        }
+    }
+
+    def construct_schema(self, class_names: typing.List[str]) -> typing.Dict[str, typing.Any]:
+        schema: typing.Dict[str, typing.Any] = MultiClassAsKeyValuePairDataset.SCHEMA_BASE  # initialize with base schema
+        schema["fieldSchema"][f"{MultiClassAsKeyValuePairDataset.CLASS_NAME_KEY}"]["classes"] = {c: {"description": f"A single class name. Only output {c} as the class name if present."} for c in class_names}
         return schema
 
-    def construct_kvp_label_data(self, label_name: str) -> typing.Dict[str, typing.Union[typing.Dict[str, typing.Dict[str, str]], None]]:
+    def construct_kvp_label_data(self, label_names: typing.List[str]) -> typing.Dict[str, typing.Union[typing.Dict[str, typing.Dict[str, str]], None]]:
         """
-        Convert the classification dataset label_name to the desired format for KVP annnotation as defined by the BASE_CLASSIFICATION_SCHEMA.
+        Convert the classification dataset label_name to the desired format for KVP annnotation as defined by the SCHEMA_BASE.
         E.g. {"fields": {"className": {"value": <label_name>}}}
 
         """
         return {
             f"{KeyValuePairLabelManifest.LABEL_KEY}": {
-                f"{CLASS_NAME_KEY}": {
-                    f"{KeyValuePairLabelManifest.LABEL_VALUE_KEY}": label_name
+                f"{MultiClassAsKeyValuePairDataset.CLASS_NAME_KEY}": {
```

Comment:
I see this `f"{variable}"` pattern in many places. Why don't we simply use the variable?